With BUILD_ALWAYS set to 1, runtimes get re-linked on install and for
some reason systemwide libclang is used, which means potential version
mismatch when looking up runtimes (e.g. 14.0.0 vs 14.0.1).

--- a/llvm/cmake/modules/LLVMExternalProjectUtils.cmake
+++ b/llvm/cmake/modules/LLVMExternalProjectUtils.cmake
@@ -321,7 +321,7 @@ function(llvm_ExternalProject_Add name source_dir)
                ${PASSTHROUGH_VARIABLES}
     INSTALL_COMMAND ""
     STEP_TARGETS configure build
-    BUILD_ALWAYS 1
+    BUILD_ALWAYS 0
     USES_TERMINAL_CONFIGURE 1
     USES_TERMINAL_BUILD 1
     USES_TERMINAL_INSTALL 1
